tests/e2e/debug-html.spec.ts

```
@@ -0,0 +1,89 @@
+      return {
+        hasNextData,
+        hasNextScripts,
+        scriptCount: scripts.length
+      }
+    })
+    
+    console.log('Next.jsマーカー:', JSON.stringify(hasHydrationMarkers, null, 2))
+    
+    // エラーを確認
+    const errors = await page.evaluate(() => {
+      return (window as any).__NEXT_DATA__?.err || null
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)